data/locale/gui/hu.po

```diff
@@ -3,807 +3,722 @@
 # This file is distributed under the same license as the lincity-ng package.
 # Copyright (C) 2024 Captain Picard <pikarkapitany@gmail.com>
 #
+#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: LinCity NG\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: Sun Aug 25 08:29:04 2024\n"
-"PO-Revision-Date: 2024-08-25\n"
+"POT-Creation-Date: Thu Sep  5 19:46:32 2024\n"
+"PO-Revision-Date: Thu Sep  5 19:46:32 2024\n"
 "Last-Translator: Captain Picard <pikarkapitany@gmail.com>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: data/gui/pbar.xml
-msgid "Population"
-msgstr "Népesség"
-
-#: data/gui/pbar.xml
-msgid "Tech Level"
-msgstr "Tech. szint"
-
-#: data/gui/pbar.xml
-msgid "Money"
-msgstr "Pénz"
-
-#: data/gui/pbar.xml
-msgid "Food"
-msgstr "Élelmiszer"
-
-#: data/gui/pbar.xml
-msgid "Labor"
-msgstr "Munkaerő"
-
-#: data/gui/pbar.xml
-msgid "Goods"
-msgstr "Áruk"
-
-#: data/gui/pbar.xml
-msgid "Coal"
-msgstr "Szén"
-
-#: data/gui/pbar.xml
-msgid "Ore"
-msgstr "Érc"
-
-#: data/gui/pbar.xml
-msgid "Steel"
-msgstr "Acél"
-
-#: data/gui/minimap.xml
+#: data/gui/minimap.xml 
 msgid "$"
 msgstr "€"
 
-#: data/gui/fire.xml
+#: data/gui/dialogs/fire.xml data/gui/dialogs/fire.xml 
 msgid "A Fire has started"
 msgstr "Tűz ütött ki."
 
-#: data/gui/shfire-uc.xml
+#: data/gui/dialogs/shfire-uc.xml 
```

Comment:
It looks like there is systematic trailing whitespace on all these lines. Maybe there's an issue with the .pot generation.